VRDR.Tests/DeathRecord_Should.cs

```diff
@@ -2115,11 +2115,53 @@ public void AgeAtDeath_RoundTrip()
             Assert.Equal("999", ije.AGE);
             Assert.Equal("9", ije.AGETYPE);
             DeathRecord dr2 = ije.ToDeathRecord();
-            Assert.Equal("", dr2.AgeAtDeath["unit"]);
-            Assert.Equal("", dr2.AgeAtDeath["value"]);
+            Assert.Equal("", dr2.AgeAtDeath["type"]);
+            Assert.Equal("", dr2.AgeAtDeath["units"]);
             Assert.True(dr2.AgeAtDeathDataAbsentBoolean);
         }
 
+        [Fact]
+        public void AgeAtDeath_EditFlag()
+        {
+            Dictionary<string, string> flag = new Dictionary<string, string>();
+            flag.Add("system", "http://hl7.org/fhir/us/vrdr/CodeSystem/vrdr-bypass-edit-flag-cs");
+            flag.Add("code", "0");
+            flag.Add("display", "Edit Passed");
+            SetterDeathRecord.AgeAtDeathEditBypassFlag = flag;
+            Assert.Equal("http://hl7.org/fhir/us/vrdr/CodeSystem/vrdr-bypass-edit-flag-cs", SetterDeathRecord.AgeAtDeathEditBypassFlag["system"]);
+            Assert.Equal("0", SetterDeathRecord.AgeAtDeathEditBypassFlag["code"]);
+            Assert.Equal("Edit Passed", SetterDeathRecord.AgeAtDeathEditBypassFlag["display"]);
+
+            flag = new Dictionary<string, string>();
+            flag.Add("system", "http://hl7.org/fhir/us/vrdr/CodeSystem/vrdr-bypass-edit-flag-cs");
+            flag.Add("code", "1");
+            flag.Add("display", "Edit Failed, Data Queried, and Verified");
+            SetterDeathRecord.AgeAtDeathEditBypassFlag = flag;
+            Assert.Equal("http://hl7.org/fhir/us/vrdr/CodeSystem/vrdr-bypass-edit-flag-cs", SetterDeathRecord.AgeAtDeathEditBypassFlag["system"]);
```

Comment:
All Profile and Extension references in CS code should use ProfileURL and ExtensionURL constants.

Reply:
There is not currently a constant defined in URLs.cs for this particular URL. Is this the incorrect URL and I need to use a different one, or is this correct and I need to add the URL (and would it fall under "profile", "extension", or "IG" urls)?

Reply:
ExtensionURL.BypassEditFlag -- 'id' of the extension
IGURL.BypassEditFlag -- 'url of documentation for the extension'.   not sure this is needed anywhere for extensions.

Reply:
It was defined in CodeSystem.cs instead of URLs.cs